h/static/styles/partials-v2/_search.scss

```diff
@@ -117,18 +114,20 @@
 }
 
 .search-result-bucket__annotation-cards {
-  flex-grow: 1;
   max-width: 500px;
+  flex-shrink: 0;
   padding-left: 0;
   padding-right: 0;
 }
 
 // Card displaying stats about a group of annotations in search results
 .search-bucket-stats {
   @include font-normal;
-  width: 210px;
   margin-left: 30px;
   word-wrap: break-word;
+  min-width: 0;  // Without this min-width stats containing really
+                 // long words won't be wrapped (word-wrap: break-word won't
+                 // work).
```

Comment:
`word-wrap: break-word` was already here but it wasn't working because `min-width: 0` is needed as well.